Saule/Http/PaginationConfig.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Configuration;
+
+namespace Saule.Http
+{
+    /// <summary>
+    /// Pagination configuration.  Property values can be set programmatically or be read from web.config app settings.
+    /// </summary>
+    public class PaginationConfig
```

Comment:
I disagree with the notion of the sensible defaults.  There's no reason to limit options for consumers of Saule, and case in point, I added this because a default of 10 wasn't appropriate for my use case.